### DISCUSSION TITLE
Tolgee and Docusaurus

### DISCUSSION THREAD

**Discussion by deployn** [tolgee/tolgee-platform#1051] [Q&A] [answered]
Hey, 
I saw Tolgee is using Docusaurus for their documentation. I tried adding Tolgee to Docusaurus (as yarn package), wasn't successful. Do you know how to connect Tolgee with Docusaurus? Is there a tutorial about that?
Best regards

**Comment by JanCizmar** [comment, marked as answer]
Since Tolgee has React integration (supporting SSR and static site generation), it should work with Docusaurus correctly. However, we haven't localized our docs yet. Is there some error message our any other information?

**Reply by deployn** [reply]
Hmm, I didn't try the React one. Will come back if this doesn't work, thank you :)

**Reply by deployn** [reply]
Unfortunately, the problem is that I can translate the pages, but I have no idea how to set up Tolgee to read and modify the documentation files.

I also don't want to convert all the Markdown documents to JSX. In that respect, I think Tolgee might not be the right tool for what I want to do.

Thanks anyway :)

**Reply by JanCizmar** [reply]
Right. Currently, we focus more on Web Apps localization. Translation of Documentation is currently out of scope. Sorry.
